README.md

```diff
@@ -1,36 +1,89 @@
+- [`bindings_js`](https://github.com/xmtp/libxmtp/tree/main/bindings_js): JS bindings (in progress)
+- [`bindings_wasm`](https://github.com/xmtp/libxmtp/tree/main/bindings_wasm): Wasm bindings (in progress)
+
+## Key features of XMTP v3-alpha
+
+### Double ratchet messaging
+
+**What**: XMTP v3-alpha introduces the use of the [double ratchet algorithm](https://signal.org/docs/specifications/doubleratchet/) to keep past and future messages secure.
+
+**Why**: With double ratchet messaging, each message is encrypted by its own session key. If the session key a sender used to send a message becomes compromised:
+
+- All of their past messages remain secure. (Forward secrecy)
+
+  Specifically, a malicious actor can’t decrypt and read the user’s past messages.
+
+- Future messages will be secure. (Future, or post-compromise, secrecy)
+
+  Specifically, a malicious actor can’t decrypt and read future messages. They also can’t send new messages impersonating the user.
+
+In contrast, with XMTP v2, if a user’s key bundle becomes compromised:
+
+- All of their past messages might not remain secure.
+- Future messages might not remain secure.
+
+**How**: Specifically, it uses the Rust version of [Olm](https://gitlab.matrix.org/matrix-org/olm/-/blob/master/docs/olm.md), which is Matrix protocol’s implementation of double ratchet. To learn more about the double ratchet algorithm and how sessions and session keys work, see [The Double Ratchet Algorithm](https://signal.org/docs/specifications/doubleratchet/).
+
+> **Tip**  
+> Use the [example CLI](https://github.com/xmtp/libxmtp/blob/main/examples/cli/README.md) to try out sending XMTP v3-alpha double ratchet messages on the XMTP `dev` network.
+
+### Installation key bundles
+
+**What**: With XMTP v2, a user signs with their blockchain account to create an XMTP identity the first time they use an app built with XMTP. The user then signs to create a [user key bundle](https://xmtp.org/docs/concepts/key-generation-and-usage) that is stored on the XMTP network and used to [encrypt](https://xmtp.org/docs/concepts/invitation-and-message-encryption) and exchange messages.
+
+With XMTP v3-alpha, there is no XMTP user key bundle. Instead, the user signs with their blockchain account to grant permission to an installation key bundle to send messages from their account address. The user signs to grant permission to an installation key bundle for each installation of an app built with XMTP. The installation key bundle is stored on each device and is not required to be stored on the XMTP network.
+
+**Why**: Storing keys on the XMTP network makes them potentially available to anyone. Moving key storage to devices significantly improves security, as keys never leave a user’s device. Additionally, installation key bundles provide separate and revocable keys for each app installation on a device, simplifying key management. When a user revokes (deletes) a key from a device, you can be sure that it is gone.
```

Comment:
```suggestion
**Why**: Storing keys on the XMTP network makes them potentially available to anyone. Moving key storage to devices significantly improves security, as keys never leave a user’s device. When a user deletes a key from a device, you can be sure that it is gone. Additionally, installation key bundles provide separate and revocable keys for each app installation on a device, simplifying key management.
```
Small nuance here - deletion and revocation are different:
- Deletion means destroying the private key for an installation, which is something you might do if you control the installation and want to clean up after yourself.
- Revocation (not implemented yet) means publishing something onto the network that declares a certain installation key is no longer valid, which is something you might do for a compromised installation on a device that you do not control.